packages/runtime/src/next/components/tests/__snapshots__/global-element-rendering.test.tsx.snap

```diff
@@ -24,7 +24,7 @@ exports[`Page correctly renders a global element 1`] = `
 }
 
 @media only screen and (min-width: 769px) {
-  .emotion-0 {
+  .emotion-1 {
```

Comment:
Update snapshots to accommodate for diffs due to the addition of the `<RootStyleRegistry>` provider, no semantic changes, just style names reshuffling.